cloudbuild.yaml

```diff
@@ -12,6 +12,7 @@ steps:
     - IMAGE_REGISTRY=gcr.io/$PROJECT_ID
     - GIT_TAG=$_GIT_TAG
     - EXTRA_TAG=$_PULL_BASE_REF
+    - DOCKER_BUILDX_CMD=/buildx-entrypoint
```

Comment:
Since `/buiild-entrypoint` does not have a `build` subcommand, we need to handle `DOCKER_BUILDX_CMD`.

https://github.com/kubernetes/test-infra/blob/cd99c042c4b16aa312f9dbee509417eea9cba9df/images/gcb-docker-gcloud/buildx-entrypoint.sh#L20

Reply:
it does:

```
$ docker run --rm -ti --privileged -v $(pwd):/workspace gcr.io/k8s-testimages/gcb-docker-gcloud:v20220830-45cbff55bc
bash-5.1# /root/.docker/cli-plugins/docker-buildx ^C
bash-5.1# /root/.docker/cli-plugins/docker-buildx build
Error: "/root/.docker/cli-plugins/docker-buildx build" requires exactly 1 argument.
```